.travis.yml

```diff
@@ -73,6 +73,16 @@ matrix:
       script:
         - make test-captplanet
     
+    ### docker build test ###
+    - env: MODE=docker
+      services:
+        - docker
+      before_script:
+      before_install:
+      install:
+      script:
+        - make all-in-one
```

Comment:
This need something that actually passes or fails. `all-in-one` just starts the environment with a satellite, a storagenode, and a gateway.

Reply:
I assumed it tests the setup?

Reply:
`all-in-one` does not test anything in a way I define test. It does stand up a local environment with production grade images.